File: Level_1/가운데 글자 가져오기/CenterStr.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int len = s.length() / 2;
        if(s.length() % 2 == 0){
            answer = s.substring(len-1, len+1);
        }else{
            answer = Character.toString(s.charAt(len));
        }
        
        return answer;
    }
}

File: Level_1/가운데 글자 가져오기/CenterStr.js
function solution(s) {
    let answer = '';
    let len = s.length / 2;
    
    if(s.length % 2 == 0){
        answer = s[len-1] + s[len];
    }else {
        len = Math.floor(len);
        answer = s[len];
    }
    
    return answer;
}
